components/radio-button/script.js

```diff
@@ -105,11 +105,33 @@ class GamefaceRadioGroup extends HTMLElement {
 
         if (!nextSibling) return;
 
-        if (nextSibling.disabled) return this.setCheckedToPreviousItem(nextSibling);
+        if (nextSibling.disabled) return this.setCheckedToNextItem(nextSibling);
         nextSibling.checked = true;
         nextSibling.focus();
     }
 
+    /**
+     * Sets the currently checked button to the previous one if it exists
+     * @returns {void}
+     */
+    checkPrev() {
+        if (this.disabled) return;
+
+        const checkedButton = this.previouslyCheckedElement;
+        this.setCheckedToPreviousItem(checkedButton);
+    }
+
+    /**
+     * Sets the currently checked button to the previous one if it exists
+     * @returns {void}
+     */
+    checkNext() {
+        if (this.disabled) return;
+
+        const checkedButton = this.previouslyCheckedElement;
+        this.setCheckedToNextItem(checkedButton);
+    }

```

Comment:
```suggestion
    checkPrev() {
        const checkedButton = this.previouslyCheckedElement;
        this.setCheckedToPreviousItem(checkedButton);
    }

    /**
     * Sets the currently checked button to the previous one if it exists
     * @returns {void}
     */
    checkNext() {
        const checkedButton = this.previouslyCheckedElement;
        this.setCheckedToNextItem(checkedButton);
    }
```

Making a check here if the group is disabled is unnecessary as the same check is done in the `setCheckedToPreviousItem` and `setCheckedToNextItem`.

Reply:
```suggestion
    checkPrev() {
        this.setCheckedToPreviousItem(this.previouslyCheckedElement);
    }

    /**
     * Sets the currently checked button to the previous one if it exists
     * @returns {void}
     */
    checkNext() {
        this.setCheckedToNextItem(this.previouslyCheckedElement);
    }

```
Simplify it even further